src/tracktor/tracker.py

```diff
@@ -411,6 +428,11 @@ def step(self, blob, frame=1):
                                     additional_dets=other_pedestrians_bboxes,
                                     early_stopping=self.finetuning_config['early_stopping_classifier']
                                 )
+                            elif self.finetuning_config["build_up_training_set"]:
```

Comment:
training samples should be added independent of whether finetune_repeatedly is activated.